src/app/application.cpp

```diff
@@ -740,9 +743,9 @@ void Application::cleanup()
 
 void Application::validateCommandLineParameters()
 {
-    if (m_commandLineArgs.portableMode && !m_commandLineArgs.profileDir.isEmpty())
+    if (m_portableModeEnabled && !m_commandLineArgs.profileDir.isEmpty())
         throw CommandLineParameterError(tr("Portable mode and explicit profile directory options are mutually exclusive"));
```

Comment:
Maybe we should remove this check completely? Because if there is profile folder in app dir and user wants to specify different profile with --profile= command line argument then app will not start at all.

Reply:
Removed.